### PR TITLE
Samples index

### PR DESCRIPTION
**Pull request by RedBrogdon**
Adds a curated list of samples to the samples repo.

### CODE REVIEW

File: INDEX.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+the Flutter team works to keep this list up to date, there are plenty of others
+created by the community, such as
+[Awesome Flutter](https://github.com/Solido/awesome-flutter) from @Solido.
+
+## How-to Collections
+
+#### [Flutter Examples](https://github.com/nisrulz/flutter-examples) [![GitHub stars](https://img.shields.io/github/stars/nisrulz/flutter-examples.svg?style=social&label=Star)](https://github.com/nisrulz/flutter-examples) [![GitHub forks](https://img.shields.io/github/forks/nisrulz/flutter-examples.svg?style=social&label=Fork)](https://github.com/nisrulz/flutter-examples/fork)
+
+A collection of single-topic examples from Nishant Srivastava. These everything
+from gradients to JSON to routing and more.
```
Comment (mjohnsullivan) [inline INDEX.md]: These 'include' everything ...

File: INDEX.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+#### [Flutter Example Apps](https://github.com/iampawan/FlutterExampleApps) [![GitHub stars](https://img.shields.io/github/stars/iampawan/FlutterExampleApps.svg?style=social&label=Star)](https://github.com/iampawan/FlutterExampleApps) [![GitHub forks](https://img.shields.io/github/forks/iampawan/FlutterExampleApps.svg?style=social&label=Fork)](https://github.com/iampawan/FlutterExampleApps/fork)
+
+Several dozen examples from Flutter GDE Pawan Kumar. These cover state
+management, Firebase, UI design, and many other topics. Each one comes with a
+YouTube videos showing implementation.
```
Comment (mjohnsullivan) [inline INDEX.md]: a YouTube 'video'

File: INDEX.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Twenty-odd samples detailing common tasks with Flutter: how to use text fields,
+streams and StreamBuilders, and more.
+
+## Architecture / networking / backend
+
+#### [Flutter Architectural Samples](https://github.com/brianegan/flutter_architecture_samples) [![GitHub stars](https://img.shields.io/github/stars/brianegan/flutter_architecture_samples.svg?style=social&label=Star)](https://github.com/brianegan/flutter_architecture_samples) [![GitHub forks](https://img.shields.io/github/forks/brianegan/flutter_architecture_samples.svg?style=social&label=Fork)](https://github.com/brianegan/flutter_architecture_samples/fork)
+
+Brian Egan's implementations of TodoMVC using a variety of state management and
+architectural approaches. If you'd like to see how an app built with
+`flutter-redux` is different than one using BLoC, this is a great place to
```
Comment (mjohnsullivan) [inline INDEX.md]: is different from - https://www.grammar.com/different-from-vs-different-than

**Comment by mjohnsullivan** [comment]
LGTM with some minor comments

File: INDEX.md (hunk 1 above)
Comment (RedBrogdon) [reply INDEX.md]: Done.

File: INDEX.md (hunk 2 above)
Comment (RedBrogdon) [reply INDEX.md]: Done.

File: INDEX.md (hunk 3 above)
Comment (RedBrogdon) [reply INDEX.md]: You can really tell I was an English major, can't you? :)

**Pull request merged**